app/frontend/src/router/index.js

```diff
@@ -20,6 +20,14 @@ const router = new VueRouter({
         requiresAuth: true
       }
     },
+    {
+      path: '/myApplications',
+      name: 'MyApplications',
```

Comment:
Should the path and name be "applications" instead?

Reply:
chaning to myApps